frontend/packages/dev-console/src/components/pipelineruns/logs/MultiStreamLogs.scss

```diff
@@ -0,0 +1,37 @@
+.odc-multi-stream-logs {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  height: 100%;
+
+  &--fullscreen {
+    .co-toolbar {
+      background-color: var(--pf-global--BackgroundColor--100);
+      padding: var(--pf-global--spacer--sm);
+    }
+  }
+  &__container {
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+    background-color: var(--pf-global--palette--black-1000);
+    color: var(--pf-global--BackgroundColor--100);
+    font-family: Menlo, Monaco, 'Courier New', monospace;
+    overflow-y: scroll;
+    position: relative;
+    &--pos {
+      position: absolute;
+      width: 100%;
+    }
+  }
+  &__taskName {
+    background-color: var(--pf-global--BackgroundColor--dark-300);
+    padding: var(--pf-global--spacer--sm) var(--pf-global--spacer--md);
+    position: sticky;
+    z-index: 99;
```

Comment:
Why this random z-index?

Reply:
removed

Reply:
This z-index is actually needed. When there is an error for logs we show an alert inside the container and while scrolling it overlaps the taskName header

Reply:
ahhh... sounds like a deeper issue. 
You should investigate that.

Reply:
I think the issue comes from the previous design of the logs (which was probably a pitfall to try and copy).

I think your layout needs to be a little more flex and a little less "hacky" with the sticky element.

```
[MultiLogs]
  [Controls]
  [Title]
  [Scrollable Container]
    [Logs.map]
```
I reordered the content and used a flex layout and it balances out nicely.

Reply:
fixed!